Sources/SpeziContact/Models/Contact.swift

```diff
@@ -11,6 +11,41 @@ import SwiftUI
 
 
 /// Encodes the contact information.
+///
+/// ### Usage
+///
+/// The following example demonstrates the usage of the ``Contact`` in a ``ContactsList``.
+/// ```swift
+/// struct Contacts: View {
```

Comment:
I'm not sure if this is a more extensive code example, it's just wrapped within a View
It would be good to see how to handle the `Contact` and what can be done with it.